lib/routes/validators.js

```diff
@@ -12,6 +12,8 @@ module.exports.HEX_STRING = /^(?:[a-fA-F0-9]{2})+$/
 // Match an encoded JWT.
 module.exports.BASE64_JWT = /^(?:[a-zA-Z0-9-_]+[=]{0,2}\.){2}[a-zA-Z0-9-_]+[=]{0,2}$/
 
+module.exports.BASE64URL = /^[a-zA-Z0-9-_]*$/
```

Comment:
naming nit: to me, `BASE64URL` sounds like it matches a url of some kind, perhaps we can call this `URLSAFEBASE64` or something that leaves less room for ambiguity.